---
costmap_topic: "/terrain/occupancy_grid"
odom_topic: "integrated_to_init"
point_cloud_topic: "velodyne_cloud_registered"


global_frame_id: "global"
is_map_frame_global: false

occupied_min_value: 100

rc_height: 0.4
spot_height: 0.75

scanVoxelSize: 0.15
inflationSize:  1 # No of cells inflated on each side.

terrainVoxelWidth: 70
quantileZ: 0.25

lowerBoundZ: -1
upperBoundZ: 0.6
min_range_y: -9.0
max_range_y: 9.0
min_range_x: -9.0
max_range_x: 9.0

disRatioZ: 0.25


# converter_plugin: "costmap_converter::CostmapToPolygonsDBSMCCH"
# # CostmapToPolygonsDBSMCCH:
# cluster_max_distance: 0.3
# cluster_min_pts: 2
# ransac_inlier_distance: 1
# ransac_min_inliers: 10
# ransac_no_iterations: 500
# ransac_remainig_outliers: 3
# ransac_convert_outlier_pts: True
# ransac_filter_remaining_outlier_pts: False
# convex_hull_min_pt_separation: 0.1
